diff --git a/node_modules/ansi-html/index.js b/node_modules/ansi-html/index.js
index 7c942d6..1f65826 100644
--- a/node_modules/ansi-html/index.js
+++ b/node_modules/ansi-html/index.js
@@ -59,7 +59,7 @@ function ansiHTML (text) {
   // Cache opened sequence.
   var ansiCodes = []
   // Replace with markup.
-  var ret = text.replace(/\033\[(\d+)*m/g, function (match, seq) {
+  var ret = text.replace(/\033\[(\d+)m/g, function (match, seq) {
     var ot = _openTags[seq]
     if (ot) {
       // If current sequence has been opened, close it.
